pages/roadmap.tsx

```diff
@@ -5,13 +5,16 @@ import {
 	getRouterRelativePath,
 	fetchRoadmap,
 	getRouterAbsolutePath,
+	useRoadmap,
 } from '@pinpt/react';
 import { PublishedRoadmapResponse } from '@pinpt/react/dist/cjs/lib/types/roadmap';
 import config from '../pinpoint.config';
 import Footer from '../components/Footer';
 import Header from '../components/Header';
 
 const Roadmap = ({ site, roadmap }: { site: ISite; roadmap: PublishedRoadmapResponse }) => {
+	const { userVotes, globalVotes, handleVote, loading, pendingVotes } = useRoadmap(config, site);
+
 	return (
 		<div>
 			<PrebuiltRoadmap
```

Comment:
need to add in Head like this one (was on my list to go through all of these today)

https://github.com/pinpt/pinpoint-changelog-app/blob/master/pages/roadmap.tsx#L14

Reply:
@jhaynie added.